hedera-mirror-rest/__tests__/specs/balances/no-params.json

```
@@ -152,6 +152,39 @@
         "hash": "c73067d7e07eaab5a6156237f928acab0a9a2783083651996237a885230a2e0f147b322fb5d0e19a0c44cf3840bd3ba9",
         "name": "1970-01-01T00_00_10.000000001Z.rcd"
       }
+    ],
+    "tokenaccounts": [
+      {
+        "token_id": "0.0.90000",
+        "account_id": "0.15.01",
```

nit, here and every other place the account is in the form of `x.y.0z`

```suggestion
        "account_id": "0.15.1",
```